app/tsconfig.json

```diff
@@ -11,13 +11,13 @@
    },
    "files": [
 	  "./src/ts/app.ts",
-    "./src/ts/upload.ts",
-    "./src/ts/play.ts",
+      "./src/ts/upload.ts",
```

Comment:
Yea........................... it works on my webstorm but I guess it may not be a permanent fix